README.md

```diff
@@ -173,7 +173,7 @@ If you're using simdjson to parse multiple documents, or in a loop, you should a
 ```c++
 // Allocate a parser big enough for all files
 document::parser parser;
-if (!parser.allocate_capacity(1024*1024)) { exit(1); }
+if (!parser.set_capacity(1024*1024)) { exit(1); }
```

Comment:
Is the argument that we basically do not need to worry about set_capacity? If so, shouldn't this be downplayed in the documentation?

Reply:
I think it should be a note for advanced users, yes. I've been trying not to change the intent of examples while I add apis, but I agree this one shouldn't call set_capacity.

By the end of this trail of prs, I believe the examples get down to a very small point, and tutorials are next.